arch/x86/adler32_ssse3.c

```diff
@@ -0,0 +1,192 @@
+/* adler32_simd.c
+ *
+ * Copyright 2017 The Chromium Authors. All rights reserved.
+ * Use of this source code is governed by a BSD-style license that can be
+ * found in the Chromium source repository LICENSE file.
+ *
```

Comment:
I emailed @Adenilson and he said to reach out via the Chromium bug tracker.

Reply:
https://bugs.chromium.org/p/chromium/issues/detail?id=1009797

Reply:
Mixing licenses is in my opinion always a bad thing due to viral nature of copyright law. If ever our code is merged with more permissive license, essentially we are giving permission for others to use the whole source using that license. On the other hand, if we merge code with more restrictive license, it might require us to remove some existing code or forbid other people linking against zlib-ng due to their code having incompatible license.

This is different if code is linked statically or dynamically as the general rule is that one binary must have only one license that covers all source files. Essentially when code is dual-licensed, the author is creating new license that combines the two licenses as such that more permissive license overrides conflicting requirements.

Reply:
The bug report has been closed. It looks like chromium developers don't see the necessity or want to relicense that bit of source code under zlib license.

Reply:
Then that code is clearly no-go...